pkgs/applications/video/screenkey/default.nix

```diff
@@ -1,55 +1,65 @@
 { lib
 , substituteAll
-, buildPythonApplication
-, fetchFromGitHub
-, distutils_extra
-, setuptools-git
+, fetchFromGitLab
+# native
 , intltool
-, pygtk
-, libX11
-, libXtst
 , wrapGAppsHook
+, file
+# not native
+, xorg
+, gobject-introspection
+, gtk3
+, python3
 , gnome3
 }:
-buildPythonApplication rec {
+
+python3.pkgs.buildPythonApplication rec {
   pname = "screenkey";
-  version = "0.9";
+  version = "1.2";
 
-  src = fetchFromGitHub {
-    owner = "wavexx";
+  src = fetchFromGitLab {
+    owner = "screenkey";
     repo = "screenkey";
-    rev = "screenkey-${version}";
-    sha256 = "14g7fiv9n7m03djwz1pp5034pffi87ssvss9bc1q8vq0ksn23vrw";
+    rev = "v${version}";
+    sha256 = "1x13n57iy2pg3h3r994q3g5nbmh2gwk3qidmmcv0g7qa89n2gwbj";
   };
 
-  patches = [
-    (substituteAll {
-      src = ./paths.patch;
-      inherit libX11 libXtst;
-    })
-  ];
-
   nativeBuildInputs = [
-    distutils_extra
-    setuptools-git
     intltool
```

Comment:
Does not look like it uses `intltool` any more: https://gitlab.com/screenkey/screenkey/-/blob/427cdc4e9043d8f9a2bcfe51df8fe9bf8600684e/po/Makefile

Reply:
Without it, I get:

```
intltool-update -p -g screenkey
unable to execute 'intltool-update': No such file or directory
error: command 'intltool-update' failed with exit status 1
builder for '/nix/store/38aq4mhha8vqmh073sgmbaypl54rq6px-screenkey-1.2.drv' failed with exit code 1
```

Reply:
Hmm, looks like they use this old thing: https://bazaar.launchpad.net/~registry/python-distutils-extra/debian/files

They should probably switch to something more modern but in the meanwhile we will have to keep this here :crying_cat_face:

Reply:
Opened https://gitlab.com/screenkey/screenkey/-/issues/122

Reply:
Awesome, I linked this issue in a comment right above `intltool`. Plus, I noticed the old patch file wasn't deleted, so I deleted in last push.